src/dd-doctor.php

```diff
@@ -42,6 +44,28 @@ function render($message, $value)
     printf('- %-42s [%s]%s', $message, $value, PHP_EOL);
 }
 
+/**
+ * Adds an indented sub-paragraph after a check to add additional hints.
+ *
+ * @param string $message
+ */
+function sub_paragraph($message)
+{
+    $wrapped = wordwrap(sprintf('  > %s%s', remove_newline($message), PHP_EOL), TEXT_WIDTH - 4, PHP_EOL . '    ');
+    printf('%s', $wrapped);
+}
+
+/**
+ * Removes new lines characters from $message and replace them with ' '.
+ *
+ * @param string $message
+ * @return string
+ */
+function remove_newline($message)
+{
+    return implode(' ', explode(PHP_EOL, $message));
```

Comment:
Just a nitpick - feel free to ignore. This might be a bit more readable :)

```suggestion
    return str_replace(PHP_EOL, ' ', $message);
```